src/context/archiveContext.js

```diff
@@ -0,0 +1,87 @@
+import { createContext, useContext } from "react"
+import axios from "axios";
+import { useNotes } from "./notesContext";
+import { useTrash } from "./trashContext";
+
+const ArchiveContext = createContext();
+
+function ArchiveProvider({ children }) {
+
+    const encodedToken = localStorage.getItem("token");
+    const { noteDispatch } = useNotes();
+    const { trashHandler } = useTrash()
+
+    const archiveHandler = async (note) => {
+        const { _id } = note;
+        try {
+            const response = await axios.post(
+                `/api/notes/archives/${_id}`,
+                {
+                    note: note,
+                },
```

Comment:
suggestion - since you're passing the object with same name it will work if write `note` instead of `note: note`. This is ES6 syntax.

Reply:
Will do.